.github/workflows/promptflow-import-linter.yml

```diff
@@ -59,4 +59,20 @@ jobs:
         cd ${{ env.WORKING_DIRECTORY }}/src/promptflow-evals
         poetry run lint-imports
       working-directory: ${{ env.WORKING_DIRECTORY }}
+    - name: import lint testing private imports from global
+      run: |
+        set -xe
+        rm ${{ env.WORKING_DIRECTORY }}/src/promptflow-tracing/promptflow/__init__.py
+        rm ${{ env.WORKING_DIRECTORY }}/src/promptflow-core/promptflow/__init__.py
+        rm ${{ env.WORKING_DIRECTORY }}/src/promptflow-devkit/promptflow/__init__.py
+        rm ${{ env.WORKING_DIRECTORY }}/src/promptflow-azure/promptflow/__init__.py
+        rm ${{ env.WORKING_DIRECTORY }}/src/promptflow-evals/promptflow/__init__.py
+        cd ${{ env.WORKING_DIRECTORY }}/src/promptflow-azure
+        echo "=== Add more import linter when facing more import errors ==="
```

Comment:
if we want import-linter detect, we need to manually add here? is it possible to apply an auto-collection for all global import from our repo?

Reply:
Yes and no.
I just suggest that we only test the ones that could be easily broken.
Some imports are embedded inside code, we might need some real statement , eg: PFClient(), to get the real import error.

Reply:
Fixed. 
![image](https://github.com/microsoft/promptflow/assets/2208599/f6ed52f9-fa2d-4b2f-8291-6300e6391830)